lua/dap/repl.lua

```diff
@@ -272,6 +273,22 @@ function M.on_down()
   select_history(1)
 end
 
+function M.prompt_backspace()
+  -- Allows backspacing through previously set text when in a prompt.
+  --
+  -- Note 1: Insert mode cursor is after (+1) the column as opposed to in normal mode it would be on (+0) the column.
+  -- Note 2: nvim_win_[get|set]_cursor is (1, 0) indexed for (line, column) while nvim_buf_[get|set]_[lines|text] is 0 indexed for both.
+
+  local currentCursor = vim.api.nvim_win_get_cursor(0)
+  local currentLineNumber = currentCursor[1]
+  local currentColumnNumber = currentCursor[2]
+  local promptLength = string.len(vim.fn['prompt_getprompt']('%'));
```

Comment:
You could use snake case to be more consistent with the rest of the code base

Reply:
good point, I'll get an update together here in a little bit